modules/serverless-sinkbinding-yaml.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// serverless/knstive_eventing/serverless-sinkbinding.adoc
+
+[id="serverless-sinkbinding-yaml_{context}"]
+= Using SinkBinding with the YAML method
+
+ This guide describes the steps required to create, manage and delete a SInkBinding instance using YAML files.
+
+.Prerequisites
+
+* You will need to have a Knative Serving and Eventing installation.
+* You will need to have the `kn` CLI installed.
+
+.Procedure
+
+. To check that SinkBinding is set up correctly, create a Knative event display service, or event sink, that dumps incoming messages to its log.
++
+Copy the following sample YAML into a file named `service.yaml`:
++
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - image: gcr.io/knative-releases/github.com/knative/eventing-contrib/cmd/event_display
```

Comment:
@abrennan89 do we want to change this to a quay.io image?

Reply:
Oops, thought I did that already... yes, and I have the one from the conversation with you and Matthias the other day :smile: thanks!